dev/com.ibm.ws.wssecurity_fat.wsscxf.1/test-applications/callerclient/src/com/ibm/ws/wssecurity/fat/callerclient/CxfCallerSvcClient.java

```diff
@@ -153,13 +148,10 @@ protected void doWorker(HttpServletRequest request,
                 service = new FatBAC02Service(wsdlURL, serviceName);
                 strExpect = "Liberty Fat Caller bac02(" + untID + ")";
 
-                //2/2021
-                if (errMsgVersion.equals("EE7")) {
-                    strSubErrMsg = "The security token could not be authenticated or authorized (Unexpected number of certificates: 0)";
-                }
-                if (errMsgVersion.equals("EE8")) {
+                //issue 23060
+                if (errMsgVersion.equals("wss4j")) {
```

Comment:
we may not need this checking done anymore here

Reply:
Will verify in the next PR.

Reply:
done in https://github.com/OpenLiberty/open-liberty/pull/23752